onnx/backend/test/case/node/reverse.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np  # type: ignore
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+class Reverse(Base):
+
+    @staticmethod
+    def export_default():  # type: () -> None
+        input = np.arange(6.0).reshape(2, 3)
```

Comment:
please specify the element data type explicitly here. and regenerate the test data to make sure we are using the right type of data.

Reply:
sure.